axum-extra/src/extract/cookie/mod.rs

```diff
@@ -84,6 +84,7 @@ pub use cookie::Key;
 ///     .route("/me", get(me));
 /// # let app: Router = app;
 /// ```
+#[must_use]
```

Comment:
```suggestion
#[must_use = "`CookieJar` should be returned as part of a `Response`, otherwise it does nothing."]
```
 This one can use a bit of clarity why it must be used if all `add` and `remove` methods and so on return `Self`.

And the same thing applies to the other jars.